feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.service.spec.ts

```diff
@@ -61,6 +61,115 @@ const orderEntry: OrderEntry = {
   ],
 };
 
+const productName0: string = 'PRODUCT_NAME_0';
+const productQty0: string = '1';
+const productFormattedPrice0: string = '$100.00';
+const productKey0: string = 'PRODUCT_KEY_0';
+const productPriceValue0: string = '100.00';
+const productName1: string = 'PRODUCT_NAME_1';
+const productQty1: string = '1';
+const productFormattedPrice1: string = '$100.00';
+const productKey1: string = 'PRODUCT_KEY_1';
+const productPriceValue1: string = '100.00';
+const v2_confInfo_version: ConfigurationInfo = {
+  configurationLabel: 'CONFIGURATION_INFO_##_VERSION',
+  configurationValue: '2',
+};
+
+const v2_confInfo0_name_novalue: ConfigurationInfo = {
+  configurationLabel: 'LINEITEM_##_0_##_NAME',
```

Comment:
We do not know which version of the OCC API is used. In the old version, label contained the Product Code, which could contain the '#' as a part. We use more complex separator to avoid this collision

Reply:
Ok, I see!

Reply:
And you don't see a necessity to customize this delimiter, right?
Could ## possibly be part of a product code?

Reply:
Our delimiter is '_##_'. Yes '##' as well as '_##_' could be a part the Product code, but using the '_##_' as a part of the product code in very unlikely. Further more we have an additional check for the version. We expect that for version 2 (or >= 2) the first configuration info line is a special one CONFIGURATION_INFO_##_VERSION. And we check this as well before processing configuration infos.

Reply:
done